scripts/pages/main-menu/main-menu.ts

```diff
@@ -2,28 +2,16 @@
 
 class MainMenu {
 	static panels = {
-		/** @type {Panel} @static */
 		cp: $.GetContextPanel(),
-		/** @type {Panel} @static */
 		pageContent: $('#PageContent'),
-		/** @type {Panel} @static */
 		homeContent: $('#HomeContent'),
-		/** @type {Panel} @static */
 		contentBlur: $('#MainMenuContentBlur'),
-		/** @type {Panel} @static */
 		backgroundBlur: $('#MainMenuBackgroundBlur'),
-		/** @type {Movie} @static */
-		movie: null,
-		/** @type {Image} @static */
-		image: $('#MainMenuBackground'),
-		/** @type {ModelPanel} @static */
-		model: $('#MainMenuModel'),
-		/** @type {Image} @static */
-		mapSelectorBackground: $('#MainMenuBackgroundMapSelectorImage'),
-		/** @type {Panel} @static */
+		movie: $('#MainMenuMovie') as Movie,
+		image: $('#MainMenuBackground') as Image,
+		model: $('#MainMenuModel') as ModelPanel,
```

Comment:
Use the C++ style template syntax that Baguettery added for stuff like this (there's more places but not gonna comment on all of them)
```suggestion
		movie: $<Movie>('#MainMenuMovie'),
		image: $<Image>('#MainMenuBackground'),
		model: $<ModelPanel>('#MainMenuModel'),
```

Reply:
Cool, this is much nicer.

Reply:
Ok applied this.